js/pages/tablespage.ts

```diff
@@ -124,6 +126,22 @@ export class TablesPage extends PageBase {
 
             return ct.nextGameTypeInformation();
         }, this);
+        this.roundNotificationCaption = ko.computed(() => {
```

Comment:
Same as above. This is very likely should be removed due to #45 and instead of make change in the UI in such way, that message DOM element would be inside TableView HTML template.